semgrep-core/tests/php/sgrep/dots_in_brackets.php

```diff
@@ -0,0 +1 @@
+$foo[...]
```

Comment:
you forgot the commit the .sgrep I think

Reply:
ah, got it, you put it in the sgrep/ directory just to check if it parses.
I think it's better when possible to make a proper .php .sgrep in tests/php/ instead.